config/examples/sandbox_proxy.yml

```diff
@@ -5,7 +5,8 @@ base: &default
   ignore_test_failures: []
   hosts:
     - localhost:8091
-  backend_host: backend:3000
+  # Backend endpoint as visible by apicast
+  backend_host: localhost:3001
```

Comment:
The name "backend" for a host looks like a container name in a docker swarm/compose network. If I understood this correctly, this is the address used from apicast to find apisonator? in that case both apicast and apisonator run as containers when we use porta-dev-tools, doesn't this break anything?

Reply:
Actually, when using porta-dev-tools, it starts APIcast with the env var `BACKEND_ENDPOINT_OVERRIDE` which overrides the setting that otherwise is configured in the APIcast config. See https://github.com/3scale-labs/porta-dev-tools/blob/8b7dc4d11224ed0d93d631a3686485a0de20385c/bin/porta#L726-L728

Reply:
I would suggest to switch to `backend_endpoint` instead of using `backend_host` + `backend_scheme`, `backend_endpoint` has preference: https://github.com/3scale/porta/blob/a0f6128f5eb16cb84aea5ce4fff74343cb98e540/app/models/proxy.rb#L324-L325

Reply:
Both very good points!

 `host.docker.internal` is used in Docker on mac and we emulate the same on Linux with podman. Latest podman also has `host.containers.internal`. So I changed this. Also changed config key.